infoschema/infoschema.go

```diff
@@ -49,6 +49,8 @@ var (
 	ErrDatabaseExists = terror.ClassSchema.New(codeDatabaseExists, "database already exists")
 	// ErrTableExists returns for table already exists.
 	ErrTableExists = terror.ClassSchema.New(codeTableExists, "table already exists")
+	// ErrTooLongIdent returns for table's name too long.
```

Comment:
The comment is not exact. It is also used for schema's and column's name too long error.